Mahmoud Mansour/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Clone</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" integrity="sha256-h20CPZ0QyXlBuAw7A+KluUYx/3pK+c7lYEpqLTlxjYQ=" crossorigin="anonymous" />
+    
+</head>
+<body>
+    <header>
+        <div class="wrapper">
+            <div class="logo">
+                <a href="https://twitter.com/home" target="_blank"><img src="./imgs/twitter_PNG13.png" alt="Twitter"></a>
+            </div>
+            <div class="btns">
+                <a href="#">Login</a>
+                <a href="#">Create New Account</a>
+            </div>
+            <div class="clr">
+
+            </div>
+        </div>
+    </header>
+    <div class="wrapper">
+    <main>
+        
+        <div class="cover">
+            <img src="./imgs/arc.jpg" alt="Stay Home">
+        </div>
+        <div class="profile-img">
+           <a href="./imgs/profile.jpg" target="_blank"><img src="./imgs/profile.jpg" alt="profile picture"></a> 
+        </div>
+        <div class="profile-detalls">
+            <h3>Mahmoud Mansour</h3>
+            <a href="https://goo.gl/maps/EJpLjbAhBrn9xVJ17"><h5><i class="fas fa-map-marker-alt icon"></i>Erbil</h5></a>
+        </div>
+        <div class="antrect-btns">
+            <a href="#">Follow</a>
+            <a href="#">Meesage</a>
+        </div>
+        <div class="spreate"></div>
+        <div class="followers">
+            <p><i class="fas fa-user"></i> <strong>12</strong> Followers you know</p>
+        </div>
+        <div class="followers-imgs">
+            <img src="./imgs/1.jpg" alt="person">
+            <img src="./imgs/2.jpg" alt="person">
+            <img src="./imgs/3.jpg" alt="person">
+            <img src="./imgs/4.jpg" alt="person">
+            <img src="./imgs/5.jpg" alt="person">
+            
+            
+            
+        </div>
+        <div class="circle">+3</div>
+        
+        
+        
+    </main>
```

Comment:
same comment as in the css 

```suggestion
            <img src="./imgs/2.jpg" alt="person">
            <img src="./imgs/3.jpg" alt="person">
            <img src="./imgs/4.jpg" alt="person">
            <img src="./imgs/5.jpg" alt="person">
        </div>
        <div class="circle">+3</div>
    </main>
```